src/app/features/cases/components/organisms/CaseTimeline/TimelineContainer.tsx

```diff
@@ -28,22 +32,38 @@ const Div = styled.div`
 }
 `
 
+const StyledButton = styled(Button)`
+  background-color: transparent;
+  @media ${ breakpoint("min-width", "laptop") } {
```

Comment:
I'm thinking of moving this positional CSS into a wrapper element inside Timeline

Reply:
Yes, that makes sense